.codeclimate.yml

```diff
@@ -1,6 +1,8 @@
 engines:
   duplication:
     enabled: true
+    exclude_fingerprints:
+    - 871d85232f90c108abf687a789b06606
```

Comment:
hmmm...should we exclude it if it's a real issue? the code climate check doesn't block builds from merging